docs/conf.py

```diff
@@ -263,3 +269,29 @@
 intersphinx_mapping = {'http://docs.python.org/': None}
 
 autoclass_content = 'both'
+
+
+# Note: For now we ignore sphinx-autodoc warnings since there are too many
```

Comment:
Yes, this is pretty darn awful, but we have too much docstring related debt right now (sadly, I don't have one week to spend fixing all those docstring related issues).

I want to start with at least fix non-docstring related issues and treating those issues as fatal going forward.